src/components/Process/Detail.tsx

```diff
@@ -0,0 +1,202 @@
+        <Text fontSize='3xl' color={'blueText'}>
+          <Trans i18nKey={'process.process_details'}>Process details</Trans>
+        </Text>
+        <ElectionSchedule
+          showRemaining
+          fontWeight={'normal'}
+          fontSize={'md'}
+          textAlign={'start'}
+          fontStyle={'normal'}
+        />
+        <ElectionSchedule
+          showCreatedAt
+          fontWeight={'normal'}
+          fontSize={'md'}
+          textAlign={'start'}
+          fontStyle={'normal'}
+        />
+      </Flex>
+      {/*Information tags */}
+      <Flex wrap={'wrap'} gap={2}>
+        {election.fromArchive && (
+          <Tag variant={'vocdoni'}>
+            <Trans i18nKey={'process.badge.archive'}>From archive</Trans>
+          </Tag>
+        )}
+        {raw.voteMode.anonymous && (
+          <Tag variant={'vocdoni'}>
+            <Trans i18nKey={'process.badge.anonymous'}>Anonymous</Trans>
+          </Tag>
+        )}
+        <Tag variant={'vocdoni'}>{censusOrigin}</Tag>
+        <Tag variant={'vocdoni'}>{autoStart}</Tag>
+        <Tag variant={'vocdoni'}>{encryptedVotes}</Tag>
+      </Flex>
+      {/*Organization card and other cards*/}
+      <Grid templateColumns={{ base: 'repeat(1, 1fr)', md: 'repeat(2, 1fr)', xl: 'repeat(4, 1fr)' }} gap={4}>
+        <GridItem colSpan={{ base: 1, md: 2 }}>
+          <OrganizationCard />
+        </GridItem>
+        <GridItem colSpan={1}>
+          <InfoCard title={t('processes.envelope_recount')}>{election.voteCount}</InfoCard>
+        </GridItem>
+        <GridItem colSpan={1}>
+          <InfoCard title={t('processes.total_questions')}>{election.questions.length}</InfoCard>
+        </GridItem>
+      </Grid>
+      {/*Encrypted votes */}
+      {raw.voteMode.encryptedVotes && <ElectionKeys electionId={id} />}
+      {/*Technical details tabs*/}
+      <Text fontSize='2xl' color={'blueText'}>
+        <Trans i18nKey={'process.detailed_data'}>Detailed data</Trans>
+      </Text>
+      <Tabs defaultIndex={defaultTab}>
+        <TabList>
+          <Tab>
+            <Trans i18nKey={'process.tab_description'}>Description</Trans>
+          </Tab>
+          <Tab>
+            <Trans i18nKey={'process.tab_results'}>Results</Trans>
+          </Tab>
+          <Tab>
+            <Trans i18nKey={'process.tab_envelopes'}>Envelopes</Trans>
+          </Tab>
+          <Tab>
+            <Trans i18nKey={'process.tab_raw'}>Raw</Trans>
+          </Tab>
+        </TabList>
+
+        <TabPanels>
+          <TabPanel>
+            <ElectionDescription />
+          </TabPanel>
+          <TabPanel>
+            <ElectionResults />
+          </TabPanel>
+          <TabPanel>
+            <p>three!</p>
+          </TabPanel>
+          <TabPanel>
+            <RawContentBox obj={raw} />
+          </TabPanel>
+        </TabPanels>
+      </Tabs>
+    </>
+  )
+}
+
+const InfoCard = ({ title, children, ...rest }: { title: string } & PropsWithChildren & CardProps) => {
```

Comment:
CardProps already contains `children`, no need for the PropsWithChildren there

```suggestion
const InfoCard = ({ title, children, ...rest }: { title: string } & CardProps) => {
```

Reply:
21fb3f9